### ISSUE TITLE
nvda Installer corrupts "nvda.exe" file when installation from portable copy

### ISSUE THREAD

**Issue by ilyapashuk** [nvaccess/nvda#17742]
### Steps to reproduce:

1. Create a portable copy of NVDA, from the installed version
2. Try to install that copy on the an other machine
3. Try to launch the installed copy

### Actual behavior:

NVDA does not launch.
the error message is "referral was returned from the server".
Then, open the Program Files\NVDA directory, and replace the "nvda.exe" from it with the nvda.exe from the portable copy. After this, it starts working.
Byte to byte compare exposes that nvda.exe from the freshly installed version is different from the nvda.exe from the portable copy.
Therefore, nvda.exe is corrupted in the installation process.

### Expected behavior:

nvda.exe file should not be modified or corrupted appon installation, it should be simply copied as is.

### NVDA logs, crash dumps and other attachments:

Attachment: 7z archive with the created portable copy and corrupted nvda.exe file

https://disk.yandex.ru/d/iYV3I8VMHhG3EA

#### NVDA installed/portable/running from source:
Portable copy is created from the installed version.
#### NVDA version:
2024.4.2
#### Windows version:
Portable copy is created on the Windows 11 24H2, launch and installation with file corruption was detected on windows 10 ltsc 1809.
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by josephsl** [comment]
Hi,

What was the NVDA version that was overridden i.e. which version was installed on the LTSC machine initially before being replaced with portable copy? You say byte-to-byte differences exist, hence my question about previous version.
Thanks.

**Comment by ilyapashuk** [comment]
Initially there were no any NVDA version on the LTSC machine.

**Comment by seanbudd** [comment]
Can you provide an NVDA log of the portable copy creating the installed copy?

**Comment by seanbudd** [comment]
Closing as we haven't heard from the author. Please provide a log and we will consider reopening it.
